static/js/components/tiles/histogram_tile.tsx

```diff
@@ -170,10 +174,20 @@ function binData(
     // Get start time in the temporal granularity desired
     const eventDate = event.startDate.slice(0, format.length);
 
-    // Increment count in corresponding bin if event has at least
+    // Get value of property to aggregate
+    let eventValue = 0;
```

Comment:
could just default to 1 and remove the else statement?

Reply:
I wrote it like this to handle the case where property="area" is set, but the corresponding displayProp was not found for the event type. In that case, all values are 0 and we won't show a broken histogram. If you think it's better to handle that case in a different way, let me know and I can switch to defaulting to 1 here.

Reply:
Gotcha, made the change!

Reply:
yes that's what I had meant!